cmake/modules/AddSwift.cmake

```diff
@@ -2128,6 +2123,17 @@ function(add_swift_target_library name)
                                    ARCHIVE DESTINATION "lib${LLVM_LIBDIR_SUFFIX}/${resource_dir}/${resource_dir_sdk_subdir}/${SWIFT_PRIMARY_VARIANT_ARCH}"
                                    COMPONENT "${SWIFTLIB_INSTALL_IN_COMPONENT}"
                                    PERMISSIONS ${file_permissions})
+      elseif(sdk STREQUAL ANDROID)
+        foreach(arch ${SWIFT_SDK_ANDROID_ARCHITECTURES})
```

Comment:
@compnerd: do you remember why you went with the primary variant architecture, instead of installing all of them? Below, around L2144 you do copy all architectures for the import libraries, for example.

There might be a way of doing all non Darwin SDKs in the same way (Linux might also benefit from multi-architectures). One can use `${sdk_supported_archs}` or `${SWIFT_SDK_${sdk}_ARCHITECTURES}` for the architecture list.